.github/workflows/build-binary.yml

```
@@ -0,0 +1,122 @@
+          --dest target
+          --name cryptomator-cli
+          --vendor "Skymatic GmbH"
+          --copyright "(C) 2016 - 2024 Skymatic GmbH"
+          --app-version "${{ needs.prepare.outputs.semVerNum }}"
+          --java-options "--enable-preview"
+          --java-options "--enable-native-access=${{ matrix.native-access-lib }}"
+          --java-options "-Xss5m"
+          --java-options "-Xmx256m"
+          --java-options '-Dfile.encoding="utf-8"'
+          ${JPACKAGE_OS_OPTS}
+      - uses: upload-artifact@v4
```

actionlint

specifying action "upload-artifact@v4" in invalid format because owner is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)